web/src/app/admin/standard-answer/StandardAnswerCreationForm.tsx

```diff
@@ -44,7 +44,7 @@ export const StandardAnswerCreationForm = ({
           }}
           validationSchema={Yup.object().shape({
             keyword: Yup.string()
-              .required("Keyword or phrase is required")
+              .required("Keywords or pattern is required")
```

Comment:
How about "Keywords" or "Regex Pattern" depending -> with a checkbox underneath that says "enable regular expression"?

Personally I would also have a field in the DB to save whether or not it's a regex match and update the logic on the inference flow to only check either keyword or regex depending.

Reply:
Sounds good. Should we also validate the regex upon submission? Might be a good user experience stopgap in lieu of a full regex composer or something similar.

Reply:
I like that, makes sense!

Reply:
Cool, I added that to this PR using a Pydantic model validator.